src/grpc_client.cc

```diff
@@ -278,12 +300,15 @@ LIBRARY_EXPORT int32_t ClientBeginClientStreamingCall(grpc_labview::gRPCid* clie
 
     auto clientCall = new grpc_labview::ClientStreamingClientCall(timeoutMs);
     *callId = clientCall;
+    clientCall->_client = client;
```

Comment:
question, the client was set for unary call already in the existing code, but here we are setting that for streaming call. Is that a bug with existing code.

Reply:
Yes. We weren't setting the _client property on the ClientCall for the streaming calls.